projects/components/src/datagrid/datagrid.component.html

```diff
@@ -1,4 +1,60 @@
+<ng-template #dropdown>
+    <clr-dropdown class="btn-group-overflow open" *ngIf="buttonConfig.contextualButtons.buttons.length !== 0">
+        <button class="btn dropdown-toggle dropdown-small" clrDropdownTrigger>
+            <clr-icon shape="ellipsis-horizontal"></clr-icon>
+        </button>
+        <clr-dropdown-menu class="dropdown-menu" *clrIfOpen>
+            <ng-container *ngFor="let button of buttonConfig.contextualButtons.buttons">
+                <button
+                    class="btn"
+                    [ngClass]="button.class"
+                    [class.disabled]="!button.shouldDisplay(getDatagridSelection())"
+                    (click)="button.handler(getDatagridSelection())"
+                >
+                    {{ button.label }}
+                </button>
+            </ng-container>
+        </clr-dropdown-menu>
+    </clr-dropdown>
+</ng-template>
+
 <clr-datagrid [clrDgLoading]="isLoading" [ngClass]="this.clrDatagridCssClass" (clrDgRefresh)="gridStateChanged($event)">
+    <clr-dg-action-bar *ngIf="buttonConfig">
+        <div class="btn-group" *ngFor="let button of buttonConfig.globalButtons">
+            <button class="btn" [ngClass]="button.class" *ngIf="button.shouldDisplay()" (click)="button.handler()">
+                {{ button.label }}
+            </button>
+        </div>
+
+        <ng-container
+            *ngIf="
```

Comment:
Avoid logic  in templates. See https://angular.io/guide/template-syntax#expression-guidelines

> A property name or method call should be the norm, but an occasional Boolean negation, !, is OK

Please fix throughout change